release/RELEASE.md

```diff
@@ -1,149 +1,157 @@
-3. A stable 2.x release.
-4. A release candidate for a 1.x release.
-5. A stable 1.x release.
-
-If you don't know which of these fit your current case, or you think it
-doesn't fit in any, please reach out on `#team-daml` on Slack, preferably
-mentioning `@gary`.
-
-## The weekly snapshot
-
-> Note: You should not have to make any change to this repo.
-
-1. On Wednesday morning, a cron should create a release PR on the [assembly]
-   repo, mentioning you. Please reach out to `@gary` on Slack if that's
-   missing.
-
-2. Merge the PR and wait for the corresponding `main` build to finish.
-
-3. Go to the [Testing](#testing) section of this file.
-
-## 2.x release candidate
-
-In a perfect world, there is no need for a separate RC: the latest weekly
-snapshot can work. In the world we live in though, we frequently want to add a
-few things, or patch previous minor releases.
-
-In those cases, we create `release/` branches (e.g. `release/2.0.x`). Those are
-special branches, protected by GitHub rules and treated specially by CI.
+   ```
+   $ git fetch
+   $ ./release.sh snapshot origin/release/2.0.x 2.0.0
+   ```
 
-When making a release candidate, you generally want to pick the tip of one of
-those release branches. The release itself is always triggered from `main`.
+   You should put that line in the file in order to preserve semver ordering,
+   and overwrite any existing snapshot with the same prefix.
 
-The process is similar to the weekly snapshot, except that both the [daml] and
-[canton] bundles have to be manually triggered. Specifically:
+   Stable releases should use the same code as the last RC, so instead of
+   adding a new line you should simply remove the `-snapshot.*` part of the
+   version string on the appropriate line of the [`LATEST`] file.
 
+2. Make a PR **targeting the `main` branch** with just that one line added,
+   touching no other file. Add the `Standard-Change` label to that PR.
 
-1. Make sure all the changes you want are in the release branch.
-2. Start _from latest `main`_ and run
-   ```
-   $ ./release.sh snapshot origin/release/2.0.x 2.0.1
-   ```
-   The output will be a line that starts with a commit sha, followed by a
-   snapshot release number. You need to take that line and add it to the [ `LATEST`]
-   file, adding ` SPLIT_RELEASE` at the end of that line. You should put that line
-   in the file in order to preserve semver ordering, and overwrite any existing
-   snapshot with the same prefix.
-3. Make a PR against the `main` branch with just that one line added, touching
-   no other file. Add the `Standard-Change` label to that PR.
 4. When the PR is merged, the build of the corresponding commit on `main` will
    create a "split release" bundle and push it to Artifactory. It should notify
    on `#ci-failures-daml` on Slack.
+
 5. The next step is to request a build from the [Canton] team (on
    `#team-canton`) that relies on the RC snapshot. Once you have a [canton]
```

Comment:
Do we want to indicate earlier that we use a dedicated channel and use that one here?

Reply:
We didn't use dedicated channels when I wrote this, but yes, if we plan to keep doing that moving forward we should mention it early on in this document.

Since you're going to make the release and likely will have a few clarifications to make on this anyway, I suggest we merge this PR as is and you open a new one with your edits next week?

Reply:
Sure. Will continue commenting on this one though.